.pre-commit-config.yaml

```diff
@@ -2,10 +2,10 @@ repos:
 - repo: https://github.com/pre-commit/pre-commit-hooks
   rev: v2.4.0
   hooks:
-  - id: trailing-whitespace
-  - id: debug-statements
-  - id: check-added-large-files
-  - id: end-of-file-fixer
+  #- id: trailing-whitespace
+  #- id: debug-statements
+  #- id: check-added-large-files
+  #- id: end-of-file-fixer
```

Comment:
Why comment out all pre-commit hooks ?

Reply:
They were segfaulting, and I was not able to make them work.
I didn't want to go down the `--no-verify` route, since the ones that don't segfault are still very useful, so I commented them. It happened to me enough times already that these hooks have become a hindrance to development, instead of being helpful, so I committed their disabling.